README.MD

```diff
@@ -13,3 +13,18 @@ For the extension to work, you must be logged in to digital ocean
 1. Navigate to where you cloned this repository
 1. Select `baby_shark`
 1. Get :moneybag: 
+
+#### Firefox Installation
```

Comment:
@the-mikedavis how about we make this `Firefox local Installation` ? Same with chrome? Once we have official packages that can go to the stores, it's no longer needed. To see what it looks like while developing we can just use `web-ext run`. Thoughts?